### ISSUE TITLE
Pasting video file results in `file:///` uri being pasted into chatbox

### ISSUE THREAD

**Issue by PandorasFox** [telegramdesktop/tdesktop#28380]
### Steps to reproduce

1. copy a video to clipboard, e.g. by Steam game recording 'copy to clipboard' clip export
2. paste into telegram
3. file:// uri is pasted as text into message box

### Expected behaviour

file should be fetched and treated as if attaching a video (other desktop applications, e.g. discord, handle this fine) or file

### Actual behaviour

text uri :(

### Operating system

windows 11

### Version of Telegram Desktop

5.5.1 x64

### Installation source

Static binary from official website

**Comment by PandorasFox** [comment]
> Can you show which url is being pasted exactly? I guess I don't treat it as a local file url.

yeah - `file:////%3F/C:/Users/me/AppData/Local/Temp/steam/2507F7DC00003.mp4` is the most recent one

**Comment by john-preston** [comment]
@PandorasFox Can you check 5.5.4 version? I've reverted a more strict check for the file urls.

**Comment by PandorasFox** [comment]
> @PandorasFox Can you check 5.5.4 version? I've reverted a more strict check for the file urls.

sure thing. on 5.5.5, it instead pops up with this error:

![image](https://github.com/user-attachments/assets/2e64ca2f-1c68-4113-b23b-0bc0aae17225)

The file in question:

![image](https://github.com/user-attachments/assets/6ebdf6d2-4ca3-40f4-a43a-d28bb7fc06b8)

i'm not entirely sure what's leading to the 'file is empty and can't be sent' since it's definitely a file with bytes in it, though

**Comment by PandorasFox** [comment]
i have since switched back to linux as my daily driver desktop OS and i can confirm that pasting videos into telegram DOES WORK - this is purely just windows nonsense. feel free to close if you want :)
